app/src/main/res/values/colors.xml

```diff
@@ -39,4 +39,5 @@
     <color name="translucent_dark_mode">#aa0a0d14</color>
     <color name="dark_mode_elevation_1">#24272e</color>
     <color name="dark_mode_semi_transparent">#e624272e</color>
+    <color name="translucent_tool_bar">#730A0D14</color>
```

Comment:
How does this colour differ from `translucent_dark_mode` (the one we used before), and should we be using that one instead?